data/json/dreams.json

```diff
@@ -503,6 +529,7 @@
     "type": "dream",
     "messages": [
       "You wonder if your nest will be good enough.",
+      "You swoop down from the skies, and somehow manage to lift off carrying an entire sheep!",
```

Comment:
This is explicitly something that can not happen in the game, let's not be intentionally misleading.
```suggestion
```